cli_reference/extending-cli-plugins.adoc

```diff
@@ -0,0 +1,17 @@
+[id='cli-extend-plugins']
+= Extending the CLI with plug-ins
+include::modules/common-attributes.adoc[]
+:context: cli-extend-plugins
+
+toc::[]
+
+You can xref:cli-writing-plugins-{context}[write] and
+xref:cli-installing-plugins-{context}[install] plug-ins to build on the default
+`oc` commands, allowing you to peform new and more complex tasks with the
+{product-title} CLI.
+
```

Comment:
I'd make a prereq at the assembly level to install and configure the CLI and xref to the assembly or assemblies.

Reply:
No, I think it's fair to have both.